performance-test/scripts/Esignet_Mockida_Test_Script.jmx

```diff
@@ -0,0 +1,1511 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="Test Plan" enabled="true">
+      <stringProp name="TestPlan.comments"></stringProp>
+      <boolProp name="TestPlan.functional_mode">false</boolProp>
+      <boolProp name="TestPlan.tearDown_on_shutdown">true</boolProp>
+      <boolProp name="TestPlan.serialize_threadgroups">false</boolProp>
```

Comment:
I think "Generate PMS Auth Token Endpoint" to be removed, because the PMS is again not in scope for eSignet with mock testing.

Reply:
1. Auth Token is generated by Generate PMS Auth Token Endpoint is passed in header file of eSignet client ID API. Hence we cant remove Generate PMS Auth Token Endpoint  . 
2. Changed from Create OIDC Client in MOSIP Authentication System (Setup) to Create OIDC Client in Mock Authentication System (Setup). And updated with eSignet API to generate client ID.

Reply:
Point 2 is fine
Point 1,  If the eSignet mock setup is enabled with authentication, we need a clientId and secret key to generate the access token. It is not necessary to the PMS client id, it can be any client id with the proper roles.
So, doing this auth token generation is fine, but lets not call that clientId and secret key as pms anywhere, since mock is not directly / indirectly dependent on PMS. (Please recheck on the authentication requirement with Anusha once again)

Reply:
We have removed Auth token generation as well as PMS dependency.